integ-test/src/test/java/org/opensearch/sql/legacy/QueryIT.java

```diff
@@ -746,6 +746,7 @@ public void dateSearch() throws IOException {
     }
   }
 
+  @Ignore
```

Comment:
Why?

Reply:
This test will fail, since he's added syntax that makes this test run against the V2 engine.

Reply:
As I understand test runs request in `json` format, so it should be executed by V1

Reply:
🔔 🔔 🔔 3 Points to Yury, the test still passes and is executed on V1, removed the `@ignore` in https://github.com/Bit-Quill/opensearch-project-sql/pull/297/commits/966efbb082b78fca450cd436641d3a3fe7a81842